src/main/fc/fc_main.c

```diff
@@ -240,7 +241,7 @@ void processRcCommand(void)
                 debug[3] = rxRefreshRate;
             }
 
-            for (int channel=0; channel < 4; channel++) {
+            for (int channel=ROLL; channel <= interpolationChannels; channel++) {
```

Comment:
I would leave strict comparison there and on #L256 and just defined `interpolationChannels` as `rcInterpolationChannels + 2`.